R/write.R

```diff
@@ -932,7 +957,7 @@ write_data_table <- function(
     na.strings      = na.strings,
     data_table      = data_table,
     inline_strings  = inline_strings,
-    dims            = dims
+    dims            = odims
```

Comment:
at this stage `dims` is the result of `dims_to_rowcol()` iirc. I overwrote whatever else was going in the code above. Might or might not be needed anymore. Dunno. The write-code is a bottomless pit and you better don't stare to long in it ...